docassemble/MLHPPOAndProposedOrder/data/questions/review.yml

```diff
@@ -216,66 +245,75 @@ review:
   - raw html: |
       ${ next_accordion('<h2 class="h5">Children</h2>') }
     show if: (len(children.complete_elements()) > 0) or (len(petitioner_other_children.complete_elements()) > 0) or (len(respondent_other_children.complete_elements()) > 0)
-  - Edit: children.revisit
+  - Edit: 
+      - children.revisit
     button: |-
       % for item in children:
 
         **${ item }**
 
-        DOB: ${ item.birthdate }
+        * DOB: ${ item.birthdate }
         % if item.lives_with:
+        % if item.lives_with == "petitioner":
+
+        * Lives with: Petitioner
+        % elif item.lives_with == "respondent":
+
+        * Lives with: Respondent
+        % elif item.lives_with == "someone_else":
 
-        Lives with: ${ item.lives_with }
+        * Lives with: Someone other than Petitioner or Respondent
+        % endif
         % endif
         % if item.parties_married_when_born:
 
-        Paternity established by ${ users[0] } and ${ other_parties[0] } being married when ${ item.familiar() } was born
+        * Paternity established by ${ users[0] } and ${ other_parties[0] } being married when ${ item.familiar() } was born
         % elif (not item.parties_married_when_born) and item.paternity == "by_court_order":
 
-        Paternity established by court order (Case \#${ item.paternity_case_number } - ${ item.paternity_case_court_name })
+        * Paternity established by court order (Case \#${ item.paternity_case_number } - ${ item.paternity_case_court_name })
         % elif (not item.parties_married_when_born) and item.paternity == "by_acknowledgment_of_parentage":
         
-        Paternity established by *Acknowledgment of Parentage*
+        * Paternity established by *Acknowledgment of Parentage*
         % elif (not item.parties_married_when_born) and item.paternity == "paternity_case_pending":
         
-        Paternity case still pending (Case \#${ item.paternity_case_number } - ${ item.paternity_case_court_name })
+        * Paternity case still pending (Case \#${ item.paternity_case_number } - ${ item.paternity_case_court_name })
         % endif
```

Comment:
I tried this out and said parties were the same gender, and that paternity was established by adoption for one child. 

The review screen block for the children did not come up because none of these are true in this case. Please add logic to display/edit the adoption case and/or legal_parent info when appropriate.

Reply:
(and check if there's any other logic specific to queer couples that might also affect the review screen)